Shared/Samples/Create convex hull around geometries/CreateConvexHullAroundGeometriesView.swift

```diff
@@ -0,0 +1,129 @@
+// Copyright 2023 Esri
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import ArcGIS
+import SwiftUI
+
+struct CreateConvexHullAroundGeometriesView: View {
+    /// A map with a topographic basemap.
+    @State private var map = Map(basemapStyle: .arcGISTopographic)
+    
+    /// The graphics overlay for the geometry graphics.
+    @State private var geometriesGraphicsOverlay: GraphicsOverlay = {
+        let polygonFillSymbol = SimpleFillSymbol(
+            style: .forwardDiagonal,
+            color: .systemBlue,
+            outline: SimpleLineSymbol(style: .solid, color: .blue, width: 2)
+        )
+        
+        // Create graphics for the example polygons.
+        let polygonGraphic1 = Graphic(geometry: .polygon1, symbol: polygonFillSymbol)
+        let polygonGraphic2 = Graphic(geometry: .polygon2, symbol: polygonFillSymbol)
+        return GraphicsOverlay(graphics: [polygonGraphic1, polygonGraphic2])
+    }()
+    
+    /// The graphics overlay for the convex hull graphics.
+    @State private var convexHullGraphicsOverlay = GraphicsOverlay()
+    
+    /// A Boolean indicating whether the button is set to create.
+    @State private var createIsOn = true
+    
+    /// A Boolean indicating whether the convex hull geometries should union.
+    @State private var shouldUnion = false
+    
+    var body: some View {
+        MapView(map: map, graphicsOverlays: [convexHullGraphicsOverlay, geometriesGraphicsOverlay])
+            .toolbar {
+                ToolbarItemGroup(placement: .bottomBar) {
+                    Toggle("Union", isOn: $shouldUnion)
+                        .toggleStyle(.switch)
+                        .onChange(of: shouldUnion) { _ in
+                            createConvexHulls()
+                        }
```

Comment:
```suggestion
```
This toggle should only set the `shouldUnion` vale right? The convex hulls are already created when the `Create` button is tapped.

Reply:
I implemented it with the idea that user could toggle the switch to see the results of unioning or not if there was already a convex hull was on the screen. Similar to the "Create buffers around points" sample, toggling shows the change without having to tap Reset and Create again. Doing so requires recreating the convex hull to show the change immediately. I updated this part a little bit implementing some of Ting's comments, but the functionality is the same. Let me know if you want the toggle to just update the shouldUnion instead.

Reply:
The updated implementation looks good :+1: